tools/heavy_measurement_lease.py

```
@@ -0,0 +1,502 @@
+STATUS_LEASE_WAITING = "lease-waiting"
+STATUS_MEASURING = "measuring"
+STATUS_COMPLETED_FINDINGS = "completed/findings"
+STATUS_COMPLETED_ZERO_FINDINGS = "completed/zero-findings"
+STATUS_CANCELLED_BEFORE = "cancelled-before-measurement"
+STATUS_INTERRUPTED_DURING = "interrupted-during-measurement"
+
+# The one status a zero claim may rest on.
+ZERO_CLAIM_STATUS = STATUS_COMPLETED_ZERO_FINDINGS
+
+DEFAULT_LEASE_PATH = os.environ.get(
+    "SUGAR_HEAVY_LEASE_PATH", "/var/tmp/sugar-heavy-measurement.lease"
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/var/tmp/sugar-heavy-measurement.lease"

(S108)